Dockerfile

```
@@ -0,0 +1,50 @@
+
+# Copy source code
+COPY . .
+
+# Build frontend
+RUN npm run build
+
+# Production stage
+FROM node:20-alpine
+
+# For HEALTHCHECK
+RUN apk add --no-cache wget
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)